### PR TITLE
[INLONG-7690][Manager] Creating schema of StreamSource by CSV

### LINKED ISSUE #7690: [Feature][Manager] Creating schema of StreamSource by CSV
### Description

Users can fill text in csv format, each row of data is separated by a line break, and each field within a row is separated by a comma or other delimiter, such as a semicolon or tab.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by featzhang**
### Prepare a Pull Request
*[INLONG-7690][Manager] Creating schema of StreamSource by CSV*

- Fixes #7690 

#### InLongStream Fields parse success

image

#### InLongStream Fields error

image

#### StreamSink Fields parse success

image

### CODE REVIEW

**Review by healchow — APPROVED** [review]
LGTM

**Review by vernedeng — APPROVED** [review]
LGTM

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java [hunk 1]
```diff
@@ -725,6 +730,41 @@ public List<SinkField> parseFields(ParseFieldRequest parseFieldRequest) {
...
+    private List<SinkField> parseFieldsByCsv(String statement) {
+        String[] lines = statement.split(InlongConstants.NEW_LINE);
+        List<SinkField> fields = new ArrayList<>();
+        for (int i = 0; i < lines.length; i++) {
+            String line = lines[i];
+            if (StringUtils.isBlank(line)) {
+                continue;
+            }
+
+            String[] cols = line.split(PARSE_FIELD_CSV_SPLITTER, 3);
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:742]: Magic number, what does **_3_** means?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java [hunk 2]
```diff
@@ -725,6 +730,41 @@ public List<SinkField> parseFields(ParseFieldRequest parseFieldRequest) {
...
+            if (cols.length < 2) {
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java:743]: ditto

**Review by vernedeng — CHANGES_REQUESTED** [review]

**Review by fuweng11 — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**